web/cobrands/sass/_layout.scss

```diff
@@ -614,6 +615,34 @@ body.authpage {
   bottom: 2em;
 }
 
+.big-hide-pins-link {
+  display: block;
+  position: absolute;
+  top: 248px; // top of #fms_pan_zoom_zoomout + height of #fms_pan_zoom_zoomout + 22px gap
```

Comment:
174+44+22 = 240, not 248?

Reply:
Haha! The extra `8px` is the `top` of `#fms_pan_zoom`, as you probably guessed. I left it out becuase the line was already getting pretty long, and I assumed nobody would care. Maybe I should add it in ;-)